default/generated/azure/tests/01-azure-aws-config.windup.test.yaml

```diff
@@ -0,0 +1,65 @@
+rulesPath: ../01-azure-aws-config.windup.yaml
```

Comment:
just to confirm, the test is against transformed yaml rules , not previous xml rules

Reply:
Yes, the test run on yaml rules in this repo